frontend/conf/upstart.conf

```diff
@@ -1,27 +0,0 @@
-# Upstart service configuration
-env USER=membership
-env USER_HOME=/membership
-
-env MANAGEMENT_PORT=9000
-export MANAGEMENT_PORT
-
-env LOGFILE="/membership/stdout.log"
-
-start on runlevel [2345]
-stop on runlevel [016]
-
-# NB: setuid is only supported on recent versions of upstart - i.e.
-#  on ubuntu not on amazon linux
-
-setuid membership
-
-chdir /membership
-
-limit nofile 65536 65536
```

Comment:
You ultimately fixed this with https://github.com/guardian/amigo/pull/94, is that right?